src/main/java/org/frc5687/steamworks/protobot/Constants.java

```diff
@@ -1,49 +1,60 @@
 package org.frc5687.steamworks.protobot;
 
-import org.frc5687.steamworks.protobot.Utils.Gamepad;
-
 /**
- * Created by Ben Bernard on 1/12/2017.
+ * Created by Admin on 1/16/2017.
  */
 public class Constants {
 
+    public static final int CYCLES_PER_SECOND = 50;
```

Comment:
If this is only used in the Limits section, consider putting this constant inside that class.